### PR TITLE
Building a lyrics search app with vanilla javascript.

### PR DESCRIPTION
**Pull request by shegz101**
### Article checklist

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #issue-number`

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by adrianmurage** [comment]
@section-engineering-education/content-moderator-team Kindly run this plagiarism check for me.
Thanks 😄

**Comment by LinusMuema** [comment]
Hi @adrianmurage & @shegz101 

Please check the files submitted. I can see there are duplicates especially in the index.md

**Comment by shegz101** [comment]
> Hi @adrianmurage & @shegz101
> 
> Please check the files submitted. I can see there are duplicates especially in the index.md

Please I do not really understand this,I thought I submitted just one index.md file.

**Comment by LinusMuema** [comment]
Check the files section of this PR

**Comment by adrianmurage** [comment]
@shegz101 

You submitted two directories: 
`building-a-lyrics-search-app-with-vanilla-javscript`
`building-a-lyrics-search-app-with-vanilla-javascript`

Kindly delete one

I will be reviewing the one with JavaScript spelled correctly

**Comment by hectorkambow** [comment]
@adrianmurage can we get a status on this PR. 

If it needs any revisions lets consider closing the PR to allow the author the proper time to implement the needed corrections.

**Comment by shegz101** [comment]
@hectorkambow this is the second PR am opening for this article, the first was closed due to some errors in the article and I have fixed the issues in this new PR

File: content/articles/building-a-lyrics-search-app-with-vanilla-javascript/index.md [hunk 1]
```diff
@@ -0,0 +1,463 @@
+
+#Building a Lyrics Search app with Vanilla Javascript
+
+![hero image](/building-a-lyrics-search-app-with-vanilla-javascript/hero.jpg)
+
+Knowing the lyrics of a song is a major problem most song-lovers encounter, especially when it is a rap song. Finding a solution to this problem, by creating a platform where users can search for lyrics by entering the artist name or title of the song is a lovely and useful project.
+
+In this article, I will walk you through how you can create a lyrics search app using HTML5, CSS3, Vanilla JS,async-await with fetch method, OVH API, and ES6 features like arrow functions.
+
+##Prerequisites
```
Comment (adrianmurage) [inline content/articles/building-a-lyrics-search-app-with-vanilla-javascript/index.md:10]: Kindly use the correct heading structure 
```suggestion
### Prerequisites
```

File: content/articles/building-a-lyrics-search-app-with-vanilla-javascript/index.md [hunk 2]
```diff
@@ -0,0 +1,463 @@
...
+..* A code editor. For example: Visual studio code.
+
+..* Some basic knowledge of HTML , CSS , and JAVASCRIPT.
+
+..* A laptop with an internet connection
```
Comment (adrianmurage) [inline content/articles/building-a-lyrics-search-app-with-vanilla-javascript/index.md:15]: Kindly use the correct list structure

File: content/articles/building-a-lyrics-search-app-with-vanilla-javascript/index.md [hunk 3]
```diff
@@ -0,0 +1,463 @@
...
+
+##Structuring The Lyrics Search App With Html5.
```
Comment (adrianmurage) [inline content/articles/building-a-lyrics-search-app-with-vanilla-javascript/index.md:17]: Kindly use the correct heading structure

File: content/articles/building-a-lyrics-search-app-with-vanilla-javascript/index.md [hunk 4]
```diff
@@ -0,0 +1,463 @@
...
+
+ To begin, create a folder named `lyrics-app`, then you create three files in the folder:
+
+..*`lyrics.html`
+..*`lyrics.css`
+..*`lyrics.js`
```
Comment (adrianmurage) [inline content/articles/building-a-lyrics-search-app-with-vanilla-javascript/index.md:23]: Kindly use the correct list structure

Though, I'm not sure whether or not this is supposed to be a list 🤔

**Review by adrianmurage — CHANGES_REQUESTED** [review]

**Comment by adrianmurage** [comment]
Hey @shegz101     👋 .

First of all, welcome to Section - EngEd. :tada: 
We are happy to have you here.

Good job on the article. It is a very interesting read.

Just a few changes I need you to make as we continue with the preliminary revisions:
- Your headings do not follow the EngEd guidelines. Refer to the [Example Article Submission](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/example_article_submission.md) for what you're supposed to do instead
- I noticed a few incorrectly formatted lists. Kindly correct those. You can refer to the [Example Article Submission](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/example_article_submission.md) for the same
- All your images including your avatar image should be scaled down to 600 x 400 px as required. You can use [Befunky](https://www.befunky.com/) to do this.

In case of any question or help needed please do not hesitate to ping me here :smile:.
Also, notify me when you are done.

Meanwhile I'll be closing this PR to clear up the PR queue. When you make all the changes and ensure your article is ready to be published. Open a new PR and ping me : )

Looking forward to reviewing and publishing your work!
Regards.

**Pull request closed without merging**